kirin/rabbitmq_handler.py

```diff
@@ -52,6 +52,7 @@ class RTReloader(ConsumerProducerMixin):
     ConsumerProducerMixin: a RPC model
     """
 
+    @new_relic.agent.background_task(name="realtime_loader-init", group="Task")
```

Comment:
To stick to code names:
```suggestion
    @new_relic.agent.background_task(name="rt_reloader-init", group="Task")
```
If it's not clear with those names, then we should change both of them to keep it matching.